game/lib/events/events_widget.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+
+import 'package:game/widget/back_btn.dart';
+import 'package:game/widget/visited_places_cell.dart';
+
+class EventsWidget extends StatefulWidget {
+  EventsWidget({Key? key}) : super(key: key);
+
+  @override
+  _EventsWidgetState createState() => _EventsWidgetState();
+}
+
+class _EventsWidgetState extends State<EventsWidget> {
+  final scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: scaffoldKey,
+      floatingActionButton: backBtn(scaffoldKey, context, "Previouly visited"),
+      backgroundColor: Color.fromARGB(255, 43, 47, 50),
+      body: Padding(
+        padding: const EdgeInsets.only(top: 150),
+        child: Container(
+          child: Padding(
+            padding: const EdgeInsets.only(left: 8.0, right: 8.0),
+            child: Column(
+              children: [
+                Container(
+                    child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Row(
+                    children: [
+                      Icon(
+                        Icons.search,
+                        size: 25,
+                        color: Colors.white,
+                      ),
+                      Expanded(
+                          child: TextField(
+                              cursorColor: Colors.white,
+                              decoration: InputDecoration(
+                                  enabledBorder: UnderlineInputBorder(
+                                      borderSide:
+                                          BorderSide(color: Colors.white)),
+                                  border: UnderlineInputBorder(),
+                                  hintText: 'search...',
+                                  hintStyle: TextStyle(
+                                      fontSize: 20, color: Colors.white),
+                                  fillColor: Colors.white)))
+                    ],
+                  ),
+                )),
+                Expanded(
+                    child: ListView(
+                        shrinkWrap: true,
+                        scrollDirection: Axis.vertical,
+                        children: [
+                      visitedPlacesCell(context, "Sage Chapel", "4/19/2021", 5,
```

Comment:
You seem to be using visitedPlacesCell here, shouldn't there be another kind of cell?

Reply:
> This might be an unfinished PR.

Yea the PR is now unfinished because I started working on a bunch of other features. I will finish them and we can merge them all after.

Reply:
Sounds good, I'll approve after that's finished.